### PR TITLE
Improve VS Code docs

### PR DESCRIPTION
**Pull request by praneesha**
Improve VS Code docs

### CODE REVIEW

File: en/docs/index.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 The Visual Studio Code Ballerina extension provides a set of rich language features along with an enhanced user experience. It offers easy development, execution, debugging, and testing for the [Ballerina programming language](https://ballerina.io/). The Ballerina language possesses a bidirectional mapping between its syntaxes and the visual representation. You can visualize the graphical representation of your Ballerina source further via the extension.
 
+For more information on the Ballerina VS Code extension, see the [VS Code marketplace](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina).
```
Comment (nilminiwso2) [inline en/docs/index.md:5]: Do we have more information about the extension in the marketplace? I thought we are bringing all that information into the docs. More information is in the documentation isn't it.

File: en/docs/get-started/install.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Install Ballerina plugin
+# Install Ballerina extension
```
Comment (nilminiwso2) [inline en/docs/get-started/install.md:1]: Shall we say `Install the Ballerina extension`

No need to drop articles IMO

File: en/mkdocs.yml [hunk 3]
```diff
@@ -15,8 +15,8 @@
 # under the License.
 
 # Project information
-site_name: Ballerina VS Code Plugin Documentation
-site_description: Ballerina VS Code Plugin Documentation
+site_name: Ballerina VS Code Extension Documentation
+site_description: Ballerina VS Code Extension Documentation
 site_author: WSO2
 site_url: https://wso2.com/ballerina/vscode/docs
```
Comment (nilminiwso2) [inline en/mkdocs.yml:22]: Shall we update the edit_uri as given below. The current URL is wrong.

```
edit_uri: https://github.com/wso2/docs-ballerina-visual-studio-code-plugin/tree/main/en/docs
```

Comment (praneesha) [reply en/mkdocs.yml:22]: Done.

File: en/docs/get-started/install.md (hunk 2 above)
Comment (praneesha) [reply en/docs/get-started/install.md:1]: Fixed.

File: en/docs/index.md (hunk 1 above)
Comment (praneesha) [reply en/docs/index.md:5]: Updated.

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**